spec/controllers/admin/schedules_controller_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+describe Admin::SchedulesController do
+  let(:conference) { create(:conference) }
+  let(:schedule) { create(:schedule, program: conference.program)}
+  let!(:organizer_role) { Role.find_by(name: 'organizer', resource: conference) }
+  let(:organizer) { create(:user, role_ids: organizer_role.id) }
+
+  context 'logged in as an organizer' do
+    before :each do
+      sign_in(organizer)
+      schedule
+    end
+
+    describe 'GET #index' do
+      it 'renders the index template' do
+        get :index, conference_id: conference.short_title
+        expect(response).to render_template :index
+      end
+    end
+
+    describe 'POST #create' do
+      it 'saves the schedule to the database' do
+        expected = expect do
+          post :create, conference_id: conference.short_title
+        end
+        expected.to change { Schedule.count }.by 1
```

Comment:
You could move it to a let

``
let(:action) { post :create, conference_id: conference.short_title } 

it 'saves the schedule to the database' do
  action.to change { Schedule.count }.by 1
end
``